src/CSI.IBTA.Administrator/Controllers/HomeController.cs

```diff
@@ -38,21 +47,75 @@ public HomeController(IUserServiceClient userServiceClient)
             if (res.Result != null)
             {
                 var employers = res.Result;
-                if (!String.IsNullOrEmpty(nameFilter))
+
+                if (!string.IsNullOrEmpty(nameFilter))
                 {
                     employers = employers.Where(s => s.Name.Contains(nameFilter));
                 }
-                if (!String.IsNullOrEmpty(codeFilter))
+                if (!string.IsNullOrEmpty(codeFilter))
                 {
                     employers = employers.Where(s => s.Code.Equals(codeFilter));
                 }
+
                 ViewData["Page"] = "Home";
-                return (View(new PaginatedList<EmployerDto>(employers ?? new List<EmployerDto>().AsQueryable(), pageNumber ?? 1, pageSize ?? 8)));
+                var employerList = employers ?? new List<EmployerDto>().AsQueryable();
+                var paginatedList = new PaginatedList<EmployerDto>(employerList, pageNumber ?? 1, pageSize ?? 8);
+                return PartialView("_Employer", paginatedList);
             }
+
             if (res.Error.StatusCode == HttpStatusCode.Unauthorized)
+            {
                 return RedirectToAction("Index", "Auth");
+            }
 
-            return View("Index");
+            return PartialView("_Employer");
+        }
+
+        [HttpGet("Claims")]
+        public async Task<IActionResult> GetClaims(
+            string? numberFilter,
+            string? employerFilter,
+            string? currentNumberFilter,
+            string? currentEmployerFilter,
+            int? pageNumber,
+            int? pageSize)
+        {
+            if (numberFilter != null || employerFilter != null)
+            {
+                pageNumber = 1;
+            }
+
+            numberFilter = numberFilter ?? currentNumberFilter;
+            employerFilter = employerFilter ?? currentEmployerFilter;
+            ViewData["CurrentNumberFilter"] = numberFilter;
+            ViewData["CurrentEmployerFilter"] = employerFilter;
+
+            var res = await _claimsClient.GetClaims();
+            if (res.Result != null)
+            {
+                var claims = res.Result;
+
+                if (!string.IsNullOrEmpty(numberFilter))
+                {
+                    claims = claims.Where(s => s.ClaimNumber.Contains(numberFilter));
+                }
+                if (!string.IsNullOrEmpty(employerFilter))
+                {
+                    claims = claims.Where(s => s.EmployerId.ToString().Equals(employerFilter));
+                }
+
+                ViewData["Page"] = "Home";
+                var claimList = claims ?? new List<ClaimDto>().AsQueryable();
+                var paginatedList = new PaginatedList<ClaimDto>(claimList, pageNumber ?? 1, pageSize ?? 8);
+                return PartialView("_Claims", paginatedList);
+            }
+
+            if (res.Error.StatusCode == HttpStatusCode.Unauthorized)
```

Comment:
I do not think this check is needed here ,as well as for other controller methods. 
If user's cookie is expired, he would be redirected to Login anyway.